src/main/java/org/testng/internal/TestListenerHelper.java

```diff
@@ -27,7 +27,9 @@ static void runPreConfigurationListeners(ITestResult tr, List<IConfigurationList
 
   static void runPostConfigurationListeners(
       ITestResult tr, List<IConfigurationListener> listeners) {
-    for (IConfigurationListener icl : listeners) {
+    List<IConfigurationListener> listeners_reverted = Lists.newArrayList(listeners);
```

Comment:
Please have `listeners_reverted` renamed to `listenersReversed`

Reply:
Done

Reply:
Done